### PR TITLE
Fix suggestions from UAT

### PR DESCRIPTION
**Pull request by vontell**
This covers quite a few UAT tasks related to the RGSample tutorial.

REG-1148 RGSample tutorial does not implement BotCharacterConfig
REG-1147 RGSample Tutorial: Missing dependencies for GameLoop.cs
REG-1146 RGSample tutorial: Missing dependencies in RGPlayerMoveAction.cs
REG-1144 RGSample tutorial should be updated to have a correct bot spawn manager implementation
REG-1142 RGSample tutorial: Update Unity scene information
REG-1141 Update screenshot of project settings
REG-1137 Indicate Unity version for RGSample tutorial

### CODE REVIEW

**Comment by vontell** [comment]
@michael-regression @rcornwall-reg if I could get your reviews this morning on this, that would be greatly appreciated!

**Review by rcornwall-reg — APPROVED** [review]
I like the new RGAction and RGState docs 👍

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 1]
```diff
@@ -100,27 +124,31 @@ _You only need to do this if you plan on interacting with the canvas_.
...
+method, [`SpawnBot()`](/studios/unity/unity-sdk/RGBotSpawnManager#public-virtual-void-seatbotbotinformation-bottospawn). 
+This method will define how our bot prefab is spawned into the scene.
 
-First, create a new file called `CharacterBotSpawnManager.cs`. It's best practice to put all Regression Games scripts
-into a common directory (we put ours in an `RGScripts/` folder).
+First, create an `RGScripts/` folder in your `Assets` folder. Then, create a file called `CharacterBotSpawnManager.cs`.
+Double click that file to open it within your editor of choice (we recommend Rider or Visual Studio). We highly recommend
+that the file is opened as part of the solution / project as a whole, so that code completion is available.
 
 Copy and paste the follow code into this file. This component allows us to drag and drop a prefab for our bots and
```
Comment (michael-regression) [inline docs/studios/unity/tutorials/first_tutorial.md:141]:
```suggestion
Copy and paste the following code into this file. This component allows us to drag and drop a prefab for our bots and
```

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 2]
```diff
@@ -134,38 +162,44 @@ public class CharacterBotSpawnManager : RGBotSpawnManager
...
+Make sure to save the scene.
 
 ![Screenshot of the RGOverlayCanvas.](first_bot_tutorial_images/tutorial_5_spawner.png)
 
 ## Add state information to relevant GameObjects
 
-The next step is to indicate what state in your scene is available to bots. This state is then used by bots to
-make decisions on what actions to take. State is relayed to bots by attaching `RGState` components to any game
-object or prefab that you want your bots to know about. 
+The next step is to indicate what state in your scene is available to bots. This state used by bots to
```
Comment (michael-regression) [inline docs/studios/unity/tutorials/first_tutorial.md:189]:
```suggestion
The next step is to indicate what state in your scene is available to bots. This state is used by bots to
```

**Review by michael-regression — COMMENTED** [review]
Everything looks good to me, suggested a couple minor grammar changes.

In `CharacterBotSpawnManager`, `SpawnBot()` isn't completely implemented. I don't think I see a place in the tutorial where the missing code is added? 

In case you want to reference it, this is the code you gave me during UAT for it:
```cs
using RegressionGames;
using RegressionGames.Types;
using UnityEngine;

public class NewBehaviourScript : RGBotSpawnManager
{
    
    [SerializeField]
    [Tooltip("The character to spawn")]
    private GameObject rgBotPrefab;

    [SerializeField]
    [Tooltip("Spawn point for RG Bots")]
    private Transform botSpawnPoint;

    public override GameObject SpawnBot(bool lateJoin, BotInformation botInformation)
    {
        Debug.Log("Spawning SAMPLE BOT");
        var bot = Instantiate(rgBotPrefab, Vector3.zero, Quaternion.identity);
        bot.transform.position = botSpawnPoint.position;

        RGPlayerMoveAction moveAction = bot.GetComponent<RGPlayerMoveAction>();
        BotCharacterConfig config = botInformation.ParseCharacterConfig<BotCharacterConfig>();
        if (config != null)
        {
            Debug.Log($"Changed speed to ${config.speed}");
            moveAction.speed = config.speed;
        }
        return bot;
        // return null;
    }
}
```

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 3]
```diff
@@ -344,9 +383,60 @@ export async function processTick(rg) {
...
+}
+return bot;
+```
+
+On bot spawn, this will grab our action script and the bot config that is being passed in, and set the speed
+to the speed defined in that JavaScript file.
+
+Start up the scene, and use the overlay to spawn a bot. While your bot is running, try updating the JavaScript 
+bot code to use very high and low values for speed. Add, commit, and push your changes, and you will see
+the bot live reload and change it's speed!
```
Comment (michael-regression) [inline docs/studios/unity/tutorials/first_tutorial.md:433]:
```suggestion
the bot live reload and change its speed!
```

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 4]
```diff
@@ -344,9 +383,60 @@ export async function processTick(rg) {
...
+
+```cs
+RGPlayerMoveAction moveAction = bot.GetComponent<RGPlayerMoveAction>();
+BotCharacterConfig config = botInformation.ParseCharacterConfig<BotCharacterConfig>();
+if (config != null)
+{
+    Debug.Log($"Changed speed to ${config.speed}");
+    moveAction.speed = config.speed;
+}
+return bot;
```
Comment (michael-regression) [inline docs/studios/unity/tutorials/first_tutorial.md:425]:
```suggestion
```
Shouldn't `return bot` already be in the function already? (So we can remove this line.) The directions also reference that the code should go before 'return bot`

**Review by michael-regression — CHANGES_REQUESTED** [review]
Just a couple minor edits, everything else looks good!

**Review by michael-regression — APPROVED** [review]

**Pull request merged**